client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -8,8 +8,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   label: {
     fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
+    color: 'rgb(0,0,0)',
     paddingLeft: '5px',
+    '&.MuiTypography-root': {
+      fontSize: 12,
+      fontWeight: 'bold',
+      marginBottom: '-1.5vh',
```

Comment:
Not sure on using viewport height for this property, could we rework this. Perhaps a new component for a custom form input and using `InputBase` and `InputLabel`